view/theme/frio/frameworks/justifiedGallery/jquery.justifiedGallery.min.js

```diff
@@ -1,8 +1,7 @@
 /*!
- * justifiedGallery - v3.8.1b
+ * justifiedGallery - v3.8.1
  * http://miromannino.github.io/Justified-Gallery/
- * Copyright (c) 2023 Miro Mannino, Hypolite Petovan
+ * Copyright (c) 2020 Miro Mannino, Hypolite Petovan, Random Penguin
```

Comment:
This seems like a Downgrade, not an upgrade o_O
2023 -> 2020 ?

Or do you mean the date of the library itself and not the last update?

Guess it's better to set 
```suggestion
 * Copyright (c) 2025 Miro Mannino, Hypolite Petovan, Random Penguin
```

what do you say?

Reply:
Sorry, I missed the different copyright date. Ignore that. This is not a downgrade. The file in Friendica was minified so I went to the Justified Gallery project GitHub and grabbed the source file, edited that, including the edit done by @MrPetovan,  and re-minified it. The file currently in Friendica is labeled as "v.3.8.1b" and I replaced it with "v.3.8.1"